src/main/java/edu/nextstep/racingcar/strategy/RacingStrategy.java

```diff
@@ -0,0 +1,7 @@
+package edu.nextstep.racingcar.strategy;
+
+public interface RacingStrategy {
+
+    public abstract boolean canMove();
```

Comment:
```suggestion
    boolean canMove();
```
abstract는 제거해도 좋을 것 같아요!